File: others/WorksApplication/Worksap Package/workspace/China_Intern/China_Intern_Java/src/jp/co/worksap/intern/entities/customer/CustomerDTO.java
package jp.co.worksap.intern.entities.customer;

import jp.co.worksap.intern.entities.ICsvMasterDTO;

public class CustomerDTO implements ICsvMasterDTO {
	/**
	 * 
	 */
	private static final long serialVersionUID = -5538243149394207296L;

	private Long customerId;
	private String name;
	private String gender;
	private String tel;

	/**
	 * Default Constructor
	 * 
	 * @param customerId
	 * @param name
	 * @param gender
	 * @param birthday
	 * @param nationality
	 * @param passportNo
	 * @param address
	 * @param email
	 * @param tel
	 */
	public CustomerDTO(Long customerId, String name, String gender, String tel) {
		super();
		this.customerId = customerId;
		this.name = name;
		this.gender = gender;
		this.tel = tel;
	}

	public Long getCustomerId() {
		return customerId;
	}

	public String getName() {
		return name;
	}

	public String getGender() {
		return gender;
	}

	public String getTel() {
		return tel;
	}

}

File: others/WorksApplication/Worksap Package/workspace/China_Intern/China_Intern_Cpp/src/entities/customer/customer_dto.cpp
#include "customer_dto.h"
#include "../../util/utilities.h"
#include <string>
using namespace std;

CustomerDTO::CustomerDTO(long customer_id, string name, string gender, string tel) {
	this->customer_id = customer_id;
	this->name = name;
	this->gender = gender;
	this->tel = tel;
}

long &CustomerDTO::get_customer_id() {
	return customer_id;
}

string &CustomerDTO::get_name() {
	return name;
}

string &CustomerDTO::get_gender() {
	return gender;
}

string &CustomerDTO::get_tel() {
	return tel;
}

ostream &operator <<(ostream &out, CustomerDTO &obj) {
	out << obj.customer_id << " " << obj.name << " " << obj.gender << " " << obj.tel;
	return out;
}

istream &operator >>(istream &in, CustomerDTO &obj) {
	in >> obj.customer_id >> obj.name >> obj.gender >> obj.tel;
	return in;
}

